src/alarms_agent.cpp

```diff
@@ -157,7 +157,21 @@ int main (int argc, char **argv)
        ii != trap_ips.end();
        ii++)
   {
-    create_trap_session(const_cast<char*>(ii->c_str()), 162, community,
+    std::string host;
+    std::string addr;
+    int port;
+
+    # Check whether there's a port specified. We've already verified it's valid 
```

Comment:
Normally comments in C should be `//` prefixed - `#` is a bizarre choice (especially as it conflicts with preprocessor macros).